src/manipulation.js

```diff
@@ -159,6 +159,10 @@ function fixInput( src, dest ) {
 }
 
 jQuery.extend({
+	htmlPrefilter: function( html ) {
```

Comment:
Thank you @dmethvin for expertly elaborating on my position. The _farthest_ I would want to go is directly exposing an array, and even for that I'd want a concrete use case—the intention here is to support permanent changes to HTML prefiltering where a user knows exactly how our default behavior is failing them and how to fix it. This is expert functionality, and experts can deal with their own registration and memory management.

[dojo/aspect](https://github.com/dojo/dojo/blob/master/aspect.js) is really nice, but tremendous overkill here.